docs/non-deterministic-error.md

```diff
@@ -65,7 +65,7 @@ In Cadence, we use "close" to describe the opposite status of “open”. For ac
 
 * Local activity is only recorded with DecisionCompleted -- no state machine needed.
 
-* Local activity completed with trigger another decision
+* Local activity completed will trigger another decision

```

Comment:
not confident this is true actually... but eh.  that's an issue for another time / a rewrite of this doc, probably.